docker/meta_requirements.txt

```diff
@@ -0,0 +1,2 @@
+mne
+moabb
```

Comment:
I think you could use a git link, to avoid waiting for a PyPi release: https://stackoverflow.com/a/35998253

Reply:
Excellent suggestion =) I loved!